### PR TITLE
Logs: Add the name of the platform we're running on to the log.

### PR DESCRIPTION
**Pull request by dragotin**

### CODE REVIEW

File: src/libsync/utility.cpp [hunk 1]
```diff
@@ -424,6 +424,21 @@ bool Utility::isBSD()
 #endif
 }
 
+QString Utility::platformName()
```
Comment (guruz) [inline src/libsync/utility.cpp]: Is there no Qt command outputting the same?

Comment (dragotin) [inline src/libsync/utility.cpp]: Thanks @guruz of course super Qt comes with something awesome. See additional patch.

**Comment by guruz** [comment]
I can 👍 this if you also add the platform to https://github.com/owncloud/client/blob/28c12a3ca0769877e7b900c5bc4b1e9148dd5abf/src/libsync/syncengine.cpp#L730 :-)

On start of sync

**Comment by dragotin** [comment]
@guruz good idea, I added stuff and now it looks like this:
```
07-28 14:45:29:966 15822 OCC::SyncEngine::startSync: "===== Using Qt 5.5.1 SSL library OpenSSL 1.0.1i-fips 6 Aug 2014 on openSUSE Leap 42.1 (x86_64)"
```

**Pull request merged**